components/clp-package-utils/clp_package_utils/scripts/native/search.py

```diff
@@ -177,18 +210,71 @@ async def do_search(
             begin_timestamp,
             end_timestamp,
             ignore_case,
-            max_num_results,
             path_filter,
-            do_count_aggregation,
-            count_by_time_bucket_size,
+            (host, port),
+            None,
+            None,
         )
     )
 
     # Wait for the job to complete or an error to occur
+    pending = [server_task, db_monitor_task]
     try:
-        await db_monitor_task
+        done, pending = await asyncio.wait(pending, return_when=asyncio.FIRST_COMPLETED)
+        if db_monitor_task in done:
+            server.close()
+            await server.wait_closed()
+        else:
+            logger.error("server_task completed unexpectedly.")
```

Comment:
There was a suggestion for a try-except here in the previous review so that we can log any error. Did you not want to add it?